app/models/project.rb

```diff
@@ -49,16 +49,18 @@ def self.data_user_projects(user)
     Project.where("(metadata_json @> ? :: jsonb) OR (metadata_json @> ? :: jsonb)", query_ro, query_rw)
   end
 
-  def approve!(session_id:, xml_namespace: nil)
```

Comment:
This will eventually be implemented via the new status field: https://github.com/pulibrary/tiger-data-app/issues/339